README.md

```diff
@@ -1,28 +1,27 @@
 # thumbor-plugins
```

Comment:
We can definitely give some more love to this readme :D Look at thumbor and thumbor-aws for inspiration?